docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -33,6 +56,8 @@ teleport:
 Restart the `teleport` process to apply the modified log level. Logs will resemble
 the following (these logs were printed while joining a server to a cluster, then
 terminating the `teleport` process on the server):
+</TabItem>
+</Tabs>
 
 ```
```

Comment:
Since these are examples of verbose logging, should we associate them with the "Updating configuration" instructions? Otherwise, they seem to be sitting here with no context.

Reply:
Both flows will change the log levels verbosity. I've moved the paragraph from below the logs example, so it gives context and then shows the example.